velox/functions/lib/window/RegistrationFunctions.h

```diff
@@ -18,7 +18,8 @@
 #include <string>
 
 namespace facebook::velox::functions::window {
-void registerBigintNthValue(const std::string& name);
-
-void registerIntegerNthValue(const std::string& name);
+void registerNthValueBigint(const std::string& name);
```

Comment:
I wonder if it is cleaner to pass TypeKind of the result as a parameter.

Please, document these functions.

Add empty line after 'namespace'.

Reply:
Added comments for these functions.